chapter3/sample_app/test/test_helper.rb

```diff
@@ -0,0 +1,14 @@
+
+ENV['RAILS_ENV'] ||= 'test'
+require_relative '../config/environment'
+require 'rails/test_help'
+require "minitest/reporters"
+Minitest::Reporters.use!
+class ActiveSupport::TestCase
+  # Run tests in parallel with specified workers parallelize(workers: :number_of_processors)
+  # Setup all fixtures in test/fixtures/*.yml for all tests in alphabetical order.
+  parallelize(workers: :number_of_processors)
+  fixtures :all
+  include ApplicationHelper
+  # Add more helper methods to be used by all tests here...
+end
```

Comment:
Please avoid pushing empty files. Thanks.

Reply:
aah I created that one in error will be more careful thank you